romsel_aktheme/arm9/source/windows/mainlist.cpp

```diff
@@ -659,8 +659,16 @@ void MainList::updateInternalNames(void)
             {
                 if (_romInfoList[_firstVisibleRowId + ii].isDSRom())
                 {
-                    _rows[_firstVisibleRowId + ii][INTERNALNAME_COLUMN]
-                        .setText(unicode_to_local_string(_romInfoList[_firstVisibleRowId + ii].banner().titles[ms().getGuiLanguage()], 128, NULL));
+					if (ms().getGuiLanguage() == 6)
```

Comment:
Does this only work for iQue games?

Reply:
> Does this only work for iQue games?

It works on every game.And the iQue games will show their Chinese Title because of the Unicode Font System from the orginal wood.